tools/ci_build/build.py

```diff
@@ -185,6 +185,8 @@ def convert_arg_line_to_args(self, arg_line):
     parser.add_argument(
         "--use_mpi", nargs='?', default=True, const=True, type=_str_to_bool)
 
+    parser.add_argument("--enable_tracy", action="store_true", help="Enable Tracy for manually performance instrument")
```

Comment:
Can we specify this via `--cmake_extra_defines` for now instead of having a dedicated argument? build.py already has masses of options, so for something like this with rare usage it's preferred to avoid adding a new argument.

Reply:
Done.